### ISSUE TITLE
[0.10.0] SELECT * from Admin UI causes panic

### ISSUE THREAD

**Issue by NorthCat** [influxdata/influxdb#5650]
Hi! I have two measurements: _download_time_measurement_ (about 1000 records) and _publication_time_measurement_ (100000+ records). Both have the same structure.
When I execute following query from **Admin UI**, all works ok: 
`SELECT * FROM "download_time_measurement";`

However, when I execute following query: 
`SELECT * FROM "publication_time_measurement"; `

my InfluxDB crashes with errors:
```
[query] 2016/02/11 13:16:05 SELECT * FROM monitoring."default".publication_time_measurement
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x63e495]
```
If I don't  use "SELECT *", all works ok:
```
SELECT SUM("articles_count") as "articles_count" , SUM("comments_count")
                as "comments_count" FROM "publication_time_measurement"  WHERE time >= '2012-02-05T14:00:00Z' GROUP BY "m_name",
                time(1d) order by time DESC;
```

I have seen these bugs: https://github.com/influxdata/influxdb/issues/2932 and https://github.com/influxdata/influxdb/issues/3016 but it seems this is not my case, because Admin UI automatically add a database parameter to URL query.

System Info:
```
InfluxDB 0.10.0
OS:
3.19.0-49-generic #55~14.04.1-Ubuntu SMP Fri Jan 22 11:24:31 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

Full log:
[log_influxdb.txt](https://github.com/influxdata/influxdb/files/127978/log_influxdb.txt)

Dump of db:
[monitoring.zip](https://github.com/influxdata/influxdb/files/127983/monitoring.zip)

**Comment by e-dard** [comment]
Hi @NorthCat. With the new query engine, the place where this panic was occurring has been refactored out/moved.

Is it easy for you to provide steps to reproduce this issue including some test data? If so, I can test it on `master` and see if it's still a problem. As it stands, I can't take your dump and use it with `master`.

**Comment by NorthCat** [comment]
Hi. @e-dard.

I downloaded and installed InfluxDB 0.9.6 (via wget) on two machines. After some time on the first machine I created a database _moitoring_ and when I executed following query 
 `SELECT * FROM "download_time_measurement";`
 got the error above. When I executed the command 
`influxdb --version`
I saw that version influx'a became **0.10.0** (I expected to see **0.9.6**). Apparently it has been updated with a system update. Today I updated to **0.10.1** by doing the following:

```
sudo apt-get update && sudo apt-get install influxdb
sudo service influxdb start
```

and the problem is gone!

Today I checked up on the second machine on which still stands Influx 0.9.6 (I have not tested it before). It is also no problem!

So, it seems everything is working fine. Thank you so much!
